kfs-core/src/main/resources/edu/arizona/kfs/fp/ojb-fp.xml

```diff
@@ -1,243 +1,352 @@
-        <foreignkey field-ref="feeIncomeChartOfAccountsCode" />
-    </reference-descriptor>
-    <reference-descriptor name="feeExpenseFinancialObject" class-ref="org.kuali.kfs.coa.businessobject.ObjectCodeCurrent" auto-retrieve="true" auto-update="none" auto-delete="none" proxy="true" >
-        <foreignkey field-ref="chartOfAccountsCode" />
-        <foreignkey field-ref="feeExpenseFinancialObjectCode" />
-    </reference-descriptor>
-    <reference-descriptor name="feeIncomeFinancialObject" class-ref="org.kuali.kfs.coa.businessobject.ObjectCodeCurrent" auto-retrieve="true" auto-update="none" auto-delete="none" proxy="true" >
-        <foreignkey field-ref="feeIncomeChartOfAccountsCode" />
-        <foreignkey field-ref="feeIncomeFinancialObjectCode" />
-    </reference-descriptor>
-    <reference-descriptor name="feeIncomeAccount" class-ref="org.kuali.kfs.coa.businessobject.Account" auto-retrieve="true" auto-update="none" auto-delete="none" proxy="true" >
-        <foreignkey field-ref="feeIncomeChartOfAccountsCode" />
-        <foreignkey field-ref="feeIncomeAccountNumber" />
-    </reference-descriptor>
-    <reference-descriptor name="clearingChartOfAccounts" class-ref="org.kuali.kfs.coa.businessobject.Chart" auto-retrieve="true" auto-update="none" auto-delete="none" proxy="true" >
-        <foreignkey field-ref="clearingChartOfAccountsCode" />
-    </reference-descriptor>
-    <reference-descriptor name="clearingAccount" class-ref="org.kuali.kfs.coa.businessobject.Account" auto-retrieve="true" auto-update="none" auto-delete="none" proxy="true" >
-        <foreignkey field-ref="clearingChartOfAccountsCode" />
-        <foreignkey field-ref="clearingAccountNumber" />
-    </reference-descriptor>
-    <reference-descriptor name="clearingFinancialObject" class-ref="org.kuali.kfs.coa.businessobject.ObjectCodeCurrent" auto-retrieve="true" auto-update="none" auto-delete="none" proxy="true" >
-        <foreignkey field-ref="clearingChartOfAccountsCode" />
-        <foreignkey field-ref="clearingFinancialObjectCode" />
-    </reference-descriptor>
-</class-descriptor>
+	<!--  CASH RECEIPT DOCUMENT MAPPINGS -->
+	<class-descriptor class="edu.arizona.kfs.fp.document.CashReceiptDocument" table="FP_CASH_RCPT_DOC_T">
+	    <!--   common transactional document base attributes -->
+	    <field-descriptor name="documentNumber" column="FDOC_NBR" jdbc-type="VARCHAR" primarykey="true" />
+	    <field-descriptor name="versionNumber" column="VER_NBR" jdbc-type="BIGINT" locking="true" />
+	    <field-descriptor name="objectId" column="OBJ_ID" jdbc-type="VARCHAR" index="true" />
+	    <field-descriptor name="nextSourceLineNumber" column="FDOC_NXT_LINE_NBR" jdbc-type="INTEGER" />
+	
+		<!-- TODO Why do we need persisting these total amounts? We never populate these fields, their getters always compute values instead of reading from DB. -->

```

Comment:
It's generally bad form to keep TODO's in newly committed code. Moreover in this case the comment no longer even applies, since these values do indeed appear to be set directly on the instance variables, and not computed as the comment claims.